scripts/unicode_lister.rb

```diff
@@ -163,18 +164,18 @@ def calculate_ranges(matching_codes)
     end
   end
     .map { |range| range.size == 1 ? range.first : range}
-    .join(", ")
 end
 
 count = 0
-File.open(OutputFilename, 'w') do |f|
+store = PStore.new("db/#{RegexpExamples::UnicodeCharRanges::STORE_FILENAME}")
+store.transaction do
   NamedGroups.each do |name|
   count += 1
     matching_codes = (0..55295).lazy.select { |x| /\p{#{name}}/ =~ eval("?\\u{#{x.to_s(16)}}") }.first(128)
-    f.puts "'#{name.downcase}' => ranges_to_unicode(#{calculate_ranges(matching_codes)}),"
+    store[name.downcase] = calculate_ranges(matching_codes)
```

Comment:
we can use [YAML store](http://ruby-doc.org/stdlib-1.9.3/libdoc/yaml/rdoc/YAML/Store.html) which entirely acts as pstore only human readable

however is slower 
you can run this to see how much

```ruby
require 'benchmark'
require 'yaml/store'
require 'fileutils'
FileUtils.rm_f('bm_blog.yml')
FileUtils.rm_f('bm_blog.pstore')
repo = YAML::Store.new('bm_blog.yml')
Post = Struct.new(:title,:body)
pstore_repo = PStore.new('bm_blog.pstore')
Benchmark.bm do |b| 
  b.report('yaml') do
    1000.times do |i| 
      repo.transaction do |r|.
        (r["posts"] ||= []) << Post.new("Post #{i}")
      end 
    end 
  end 
  b.report('pstore') do
    1000.times do |i| 
      pstore_repo.transaction do |r|.
        (r["posts"] ||= []) << Post.new("Post #{i}")
      end 
    end 
  end 
end
```
(stolen from Avdi Grim ruby tapas 163)